pkg/channel/http/http_channel.go

```diff
@@ -61,6 +61,7 @@ func CreateLocalChannel(port, maxConcurrency int, spec config.TracingSpec, sslEn
 		client: &fasthttp.Client{
 			MaxConnsPerHost:           1000000,
 			MaxIdemponentCallAttempts: 0,
+			MaxResponseBodySize:       maxRequestBodySize * 1024 * 1024,
```

Comment:
This is a limit on the response size that gets delivered from the server, probably not what you want here?

Reply:
true let me revert that change.

Reply:
@yaron2 Fixed.